src/components/PassageRecord.tsx

```diff
@@ -217,9 +218,9 @@ function PassageRecord(props: IProps) {
           onClick={handleAddOrSave}
           variant="contained"
           color="primary"
-          disabled={(ready && !ready()) || !filechanged || name === ''}
+          disabled={(ready && !ready()) || name === ''} //|| !filechanged}
```

Comment:
should this comment just be removed?

Reply:
I really didn't want to pick through what I was still working on to get the fix for Michael checked in.  I'm still working on verifying that saving with different file types actually does something.